ls1intum/Artemis
# [Bugfix] Fix incorrect repository link

### Checklist

### Motivation and Context



This PR is related to the Bug #1860

The implemented solution is a quick fix, that removes the username from the `repositoryURL`. At the moment I do not understand the system good enough to say, if the `repositoryURL` needs to be stored this way. The quick fix should work for TumIDs because they don't consist of `@`

Original PR by @siboMuc (#2275)

### Description


- Use the adjustedRepositoryUrl, which is created without the username in the url

### Steps for Testing


1. Log in to Artemis as an instructor
2. Navigate to the manual assessment page for a programming participation
3. Click on the 'Go to repository' link
4. The correct bitbucket / gitlab url should be opened in a new tab.

### Test Coverage




* code-editor-tutor-assessment-container.component.ts: **84.4% stmts**

### Screenshots



<img width="509" alt="Bildschirmfoto 2020-11-11 um 16 52 59" src="https://user-images.githubusercontent.com/13920539/98833488-63733900-243e-11eb-8a82-3c3d2da2bc31.png">
